sanic/server.py

```diff
@@ -189,9 +190,15 @@ def write_error(self, exception):
                 "Writing error failed, connection closed {}".format(e))
 
     def bail_out(self, message):
-        exception = ServerError(message)
-        self.write_error(exception)
-        log.error(message)
+        if self.transport.is_closing():
+            log.error(
+                "Connection closed before error was sent to user @ {}".format(
+                    self.transport.get_extra_info('peername')))
```

Comment:
Well in the example that we had seen from the issue self.parser was still None so I'd be wary of if self.request was filled out correctly as well. It's just a safe way of doing it.